src/cmd-init

```diff
@@ -6,6 +6,8 @@ dn=$(dirname $0)
 
 preflight
 
+sudo setfacl -d -m u:builder:rwX .
```

Comment:
Why do we need this one?

Reply:
I'm assuming the user created the initial directory as root and won't have provided access to the builder user.